views/search.erb

```diff
@@ -0,0 +1,4 @@
+<form action="/filtered" method="post">
```

Comment:
This would be really cool if this page was on your homepage, so the search function is available from the homescreen? Really cool that you got it to work though!